worlds/sm64ex/Regions.py

```diff
@@ -5,8 +5,9 @@
     locHMC_table, locLLL_table, locSSL_table, locDDD_table, locSL_table, \
     locWDW_table, locTTM_table, locTHI_table, locTTC_table, locRR_table, \
     locPSS_table, locSA_table, locBitDW_table, locTotWC_table, locCotMC_table, \
-    locVCutM_table, locBitFS_table, locWMotR_table, locBitS_table, locSS_table    
+    locVCutM_table, locBitFS_table, locWMotR_table, locBitS_table, locSS_table
 
+# List of all courses, including secrets, without BitS as that one is static
```

Comment:
Comment is at a weird place. Currently looks like it is just referencing `sm64_level_to_paintings`, which does not include secrets.
In the first place these arent lists, but dicts from the game-specific entrance format to the name of the entrance.
Does make me realize I made the same mistake just below for sm64secrets though ^^'

Reply:
Removed that comment and cleaned up the secrets comment.